code/modules/projectiles/guns/energy/cell_loaded_vr/multi_cannon.dm

```diff
@@ -0,0 +1,69 @@
+//Why is this in here when it's not a subtype of cell_loaded? Because it has a similar function, and I couldn't be assed to find a better suited spot.
+
+/obj/item/weapon/gun/projectile/multi_cannon
+	name = "Curabitur Cannon"
+	
+	desc = "A cannon developed by Curabitur Scimed, this weapon incorporates both Vey-Med and precursor technology to create a medical alternative to chemicals on the field."
+	icon = 'icons/vore/custom_guns_vr.dmi'
+	icon_state = "healcannon"
+
+	description_info = "Created to fulfill the needs and wants of the Curabitur Scimed personell during rescue operations, the Healcannon is a marvel of reverse-engineering and utilization of unknown technologies. \
+	It makes use of the Vey-Med ML-3 'Medigun' microbattery technology, combining the effects of multiple 'microbatteries' into single 'macrobatteries' that are built around precursor void cores, \
+	allowing for the batteries to self-charge for prolonged field use. However, the weakened beams caused by the use of ununderstood technology created a need for a VERY strong focus, \
+	which lead to the Healcannon becoming a very bulky tool. Fortunately, it's not as heavy as it looks." 
+
+	w_class = ITEMSIZE_LARGE
+	load_method = SINGLE_CASING
+	max_shells = 1
+
+	handle_casings = HOLD_CASINGS
+	auto_eject = FALSE
+
+
+	caliber = "macrobat"
+
+
+	icon_override = 'icons/vore/custom_guns_vr.dmi'
+	item_state = "healcannon"
+
+/obj/item/weapon/gun/projectile/multi_cannon/update_icon()
+	. = ..()
+	overlays.Cut()
```

Comment:
```suggestion
	cut_overlays()
```

So you can use SSoverlays to cut the overlays rather than asking byond to do it directly.